homeassistant/components/tankerkoenig/__init__.py

```diff
@@ -21,7 +21,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-DEFAULT_RADIUS = 2
+DEFAULT_RADIUS = 5
```

Comment:
This is sort of a breaking change. The alternative is to update the docs. 🤷‍♂️

Reply:
I will update the docs then 😄

Reply:
Originally I had 5 as default (and that's what I wrote in the docs). Then in the pull request someone mentioned that 5 Km. might be too big in cities, to which I agreed, and changed the default to 2 Km.
Unfortunately, I forgot to update the documentation as well :-(

@springstan, no need to do the docu PR, I just did:
https://github.com/home-assistant/home-assistant.io/pull/12361

Reply:
@guillempages thanks for taking care of it 👍

Reply:
Well, I broke it, so I fix it ;-)

Reply:
That is a good attitude ✌